velox/exec/tests/TableWriteTest.cpp

```diff
@@ -2262,6 +2264,114 @@ TEST_P(AllTableWriterTest, tableWriteOutputCheck) {
   ASSERT_EQ(obj[TableWriteTraits::kLifeSpanContextKey], "TaskWide");
 }
 
+TEST_P(AllTableWriterTest, columnStatsDataTypes) {
```

Comment:
```
s/columnStatsDataTypes/columnStatsVerification/
```

Reply:
we have column stats test underneath that verify the response data layout, this test focuses on different data types, so I think it's better to keep this name